### DISCUSSION TITLE
Magefan Magento 2 WebP Images Extension

### DISCUSSION THREAD

**Discussion by SIDRONA73** [breezefront/community#71] [3. Integrations]
Would like help with third-party modules integrations with Breeze module.

. Magefan Magento 2 WebP Images Extension

Thank you for any help!

**Comment by dimdimich** [comment]
Thank you for using Breeze. I would suggest to follow next steps described at https://breezefront.com/extensions

First step to do is to ask your extension vendor to add compatibility. Considering logic of WEBP module i would expect that it would work either right out of the box or it will require minimal integration.

**Comment by magefan** [comment]
The Magefan WebP Extension for Magento works with Breeze an other popular third-party themes out of the box. If you face any issues, feel free to contact our support team right away.
